queries/queryClient.ts

```diff
@@ -1,12 +1,16 @@
 import { QueryClient } from "@tanstack/react-query";
-import { GC_TIME, STALE_TIME } from "./queryClient.constants";
+import { DEFAULT_GC_TIME, DEFAULT_STALE_TIME } from "./queryClient.constants";
 
 export const queryClient = new QueryClient({
   defaultOptions: {
     queries: {
       retry: false, // libXmtp handles retries
-      gcTime: GC_TIME,
-      staleTime: STALE_TIME,
+      retryOnMount: false, // Prevent infinite refetch loops by manually controlling when queries should retry on component mount
```

Comment:
nit when we do have comments can we put them on top not to the side of the code?